my-app/src/Board.js

```diff
@@ -0,0 +1,88 @@
+import React, { useReducer } from 'react';
+import logo from './logo.svg';
+import Square from './Square';
+import {
+  BoardSquareContext,
+  BoardSetSquareContext,
+  SquareContext,
+} from './App.js';
+
+// main board component
+function Board() {
+  const boardSquares = React.useContext(BoardSquareContext);
+  const setBoardSquares = React.useContext(BoardSetSquareContext);
+
+  const initialSate = 'true';
+  const reducer = (state, action) => {
+    switch (action) {
+      case 'setXPlayer':
+        return !state;
+    }
+  };
+  const [isXPlayer, dispatch] = useReducer(reducer, initialSate);
```

Comment:
Nice - I like that you are using `useReducer` here as well, because you are demonstrating knowledge of different pieces of React. For simple pieces of state like this one though, I'd probably prefer `useState` since it's a little bit less complex. `useReducer` could be a good choice if you wanted to hoist all of this logic onto the context in the parent component.